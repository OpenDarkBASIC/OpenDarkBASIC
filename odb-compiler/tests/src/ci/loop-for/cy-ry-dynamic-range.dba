for n = mul(1) to mul(5) step 1
    printc stdout n
next n
end

function mul(value as integer)
endfunction value * 2
