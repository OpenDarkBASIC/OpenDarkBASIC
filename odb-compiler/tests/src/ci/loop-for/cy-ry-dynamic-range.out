"2345678910\n"
